### ISSUE TITLE
The Cipher of Damnation - Borak's Charge

### ISSUE THREAD

**Issue by crazyyyyyyyyyyy** [WoWManiaUK/Redemption#5622]
**Links:**
https://tbc.wowhead.com/quest=10578/the-cipher-of-damnation-boraks-charge

**What is Happening:**
Players should kill Ruul and his dragon https://tbc.wowhead.com/npc=21315/ruul-the-darkener in order to get the https://tbc.wowhead.com/item=30645/third-fragment-of-the-cipher-of-damnation for completing the quest

**What Should happen:**
Once I've killed Ruul it doesn't drop the quest item and Ruul doesn't also respawns in the quest location.

**Comment by crazyyyyyyyyyyy** [comment]
He spawns in the wrong side of the map

**Comment by Rushor** [comment]
can you explain more detailed?

**Comment by JormagMania** [comment]
Good day,

I did this quest last night, December 11th 2022 and I concur with an issue in pathing. Ruul spawns somewhere and is supposed to fly down to the ground in the Netherwing Field but he did not. After flying around a bit, I have found Ruul, stuck in the air at what seems to be his spawn location. 56.9, 49.6 coordinate. 

![Screenshot (3519)](https://user-images.githubusercontent.com/120397554/207103870-f19892a6-4f5c-46ee-9879-cee72f335690.png)

![Screenshot (3518)](https://user-images.githubusercontent.com/120397554/207103906-c12152fb-6406-4cd0-aa3b-7b97e4484837.png)


I was able to aggro him from my flying mount and land on the top of a tower to kill him and loot the third fragment of the cipher as intended.  It seems that his pathing is not working and he is not landing as intended at the quest location marked on the map. 

![Screenshot (3520)](https://user-images.githubusercontent.com/120397554/207103946-16e90c8d-a79e-4f96-b6a0-26ea7bc0211b.png)


Thank you.

Jormag.

 He did drop the quest item not like mentioned above. Only the pathing is not working.
